New approach, making feature from scratch again

123rvb123 opened the pull request:
Making the login/logout/account feature from scratch again, currently I am able to make new users and store them into the mongodb server, and can log in with this account. Need to do a few more things such as using passport and converting from ejs to handlebars

123rvb123 committed:
New approach, making feature from scratch again

Making the login/logout/account feature from scratch again, currently I am able to make new users and store them into the mongodb server, and can log in with this account. Need to do a few more things such as using passportand converting from ejs to handlebars

123rvb123 committed:
Passport topic added

Passport is succesfully added, authentication is up. Now I still have to convert the ejs to handlebars to match the petscout project, and edit a lot for the petscout project.

123rvb123 committed:
Converted most html/css to our project petscout

Used bootstrap to just have some quick html and css for the feature, but since the feature works I converted everything to HTML & CSS for our petscout project!

Everything looks good just needs some finishing touches like changing the text a bit and design the error messages a bit more nicely

CesarvanH reviewed (commented):
je kan hem beter naar dev mergen want main is zegmaar de holy grail

Falicer reviewed (changes requested):
Meerdere dingen:
1. Het moet eerst gemerged worden met de dev branch
2. Waarom staat er dat de config, routing en schematics gedelete worden?

123rvb123 committed:
Changed some things like css

A few changes on the css and old bootstrap things I had in the documents, also removed a few things

123rvb123 commented:
Oh oops, was niet mn bedoeling om ze naar main te mergen

123rvb123 committed:
Converting from ejs to handlebars

So far the pages have been loaded and everything works, I just have to output messages as handlebars still
